FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,47 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+		client.get(from: url) { [weak self] result in
+			if self == nil { return }
```

Comment:
A `guard` statement is preferred to validate the state and exit early (since it requires the return statement!):

```suggestion
			guard self != nil else { return }
```